core/qthelper.h

```diff
@@ -116,6 +116,19 @@ void moveInVector(Vector &v, int rangeBegin, int rangeEnd, int destination)
 		std::rotate(it + destination, it + rangeBegin, it + rangeEnd);
 }
 
+// Qt overloads some signals(!) which can't therefore be used in connect() calls with
+// pointers-to-member functions where the signatures of signal and slot don't match perfectly.
+// For this case, Qt 5.7 provides the qOverload<> helper.
+// Since we still support Qt 5.5 let's reimplement it here.
+template <typename... Args>
+struct qOverload {
+	template <typename Ret, typename Class>
+	static Ret (Class::*of(Ret (Class::*fun)(Args...)))(Args...)
+	{
+		return fun;
+	}
+};
```

Comment:
so why doesn't this conflict on Qt5.7 and newer?
I am so confused...

Reply:
Because typo (qOverload vs. QOverload). Will change once I find time...